src/types/eventTypes.ts

```diff
@@ -71,3 +71,9 @@ export interface SlimPopup {
   description: BlocksContent
   text_content?: BlocksContent
 }
+
+export interface DigestedEvent {
+  id: number
+  title: string
+  pop_ups: Favourite[]
+}
```

Comment:
I don't think we need a new type for this. This is just EventData interface minus a couple of fields.